src/server/scan/listener/helpers/process-scan-complete.js

```diff
@@ -1,36 +1,36 @@
+import { config } from '~/src/config'
 import {
   isReady,
   isUploadPending,
   uploadStatus
 } from '~/src/server/common/helpers/upload-status'
+import { fileStatus } from '~/src/server/common/constants/file-status'
 import { sendSqsMessage } from '~/src/server/common/helpers/sqs/send-sqs-message'
-import { isInfected } from '~/src/server/common/helpers/file-status'
-import { config } from '~/src/config'
 import { createUploadLogger } from '~/src/server/common/helpers/logging/logger'
 
 const callbackQueueUrl = config.get('sqsScanResultsCallback')
-async function processScanComplete(server, uploadId, fileId) {
-  const { files, uploadDetails } =
-    await server.redis.findUploadAndFiles(uploadId)
 
+async function processScanComplete(server, uploadId) {
+  const uploadAndFiles = await server.redis.findUploadAndFiles(uploadId)
+  const files = uploadAndFiles?.files
+  const uploadDetails = uploadAndFiles?.uploadDetails
   const uploadLogger = createUploadLogger(server.logger, uploadDetails)
   const isPending = isUploadPending(uploadDetails.uploadStatus)
 
   if (isPending && isScanningComplete(files)) {
     uploadDetails.uploadStatus = uploadStatus.ready.description
-    uploadDetails.ready = new Date()
-    uploadDetails.numberOfInfectedFiles = numberOfInfectedFiles(files)
-    await server.redis.storeUploadDetails(uploadId, uploadDetails)
+    uploadDetails.ready = new Date().toISOString()
+    uploadDetails.numberOfRejectedFiles = numberOfRejectedFiles(files)
 
-    const readyUploadLogger = createUploadLogger(server.logger, uploadDetails)
-    readyUploadLogger.info(`Upload marked as ready`)
+    await server.redis.storeUploadDetails(uploadId, uploadDetails)
+    server.logger.info({ uploadDetails }, 'Upload marked as ready')
 
     if (uploadDetails.scanResultCallbackUrl) {
       try {
         await sendSqsMessage(server.sqs, callbackQueueUrl, { uploadId })
       } catch (error) {
-        readyUploadLogger.error(
-          error,
+        server.logger.error(
+          { uploadDetails, error },
```

Comment:
I don't think this will work as uploadDetails is now nested

Reply:
This logs as intended

<img width="976" alt="Screenshot 2024-04-30 at 09 36 15" src="https://github.com/DEFRA/cdp-uploader/assets/2305016/ad6f0a49-766e-44ff-ad57-ad14b83f8ced">

Reply:
I don't think this is as expected - we don't want to log uploadDetails as it can contain PII. lets discuss in person